enzyme/test/ActivityAnalysis/integration.ll

```diff
@@ -0,0 +1,133 @@
+; RUN: if [ %llvmver -lt 16 ]; then %opt < %s %loadEnzyme -print-activity-analysis -activity-analysis-func=f.preprocess -o /dev/null | FileCheck %s; fi
```

Comment:
Since this uses opaque pointers you'll probably need to do an appropriate version gate here